sources/packages/web/src/views/institution/designations/DesignationRequest.vue

```diff
@@ -1,13 +1,12 @@
 <template>
-  <v-container>
-    <v-row justify="center">
-      <div class="pb-4 w-100 full-page-container-size">
-        <h5 class="text-muted">Back to manage designation</h5>
-        <h2 class="category-header-large">Request designation</h2>
-      </div>
-    </v-row>
-  </v-container>
-  <full-page-container>
+  <full-page-container :full-width="true">
```

Comment:
Can you please elaborate which is the impact of `full-width` in the UI?

Reply:
`full-width= true` will set `fluid=true` for the container, which will utilize the whole width